src/Schema/Values/ContextModule.ts

```diff
@@ -96,6 +96,7 @@ export enum ContextModule {
   popUpModal = "popUpModal",
   presentingFair = "presentingFair",
   presentingPartner = "presentingPartner",
+  priceDatabaseLanding = "priceDatabaseLanding",
```

Comment:
feedback welcome on the context module name, but I thought this summed up the general idea for where users will be searching